src/command/restore/restore.c

```diff
@@ -2073,6 +2074,32 @@ restoreProcessQueue(const Manifest *const manifest, List **const queueList)
             const ManifestFilePack *const filePack = manifestFilePackGet(manifest, fileIdx);
             const ManifestFile file = manifestFileUnpack(manifest, filePack);
 
+            Oid dbNode, relNode;
+            if (sscanf(strZ(file.name), "pg_data/base/%u/%u", &dbNode, &relNode) == 2)
+            {
+                if (!isRelationNeeded(dbNode, DEFAULTTABLESPACE_OID, relNode))
+                    continue;
+            }
+            else if (strBeginsWithZ(file.name, "pg_tblspc/"))
```

Comment:
Directly such lines are used only in tests. Try using ready-made macros like https://github.com/arenadata/pgbackrest/blob/3a86d4ea84fec499148cebf9144344d34de24fd8/src/command/restore/restore.c#L1285-L1286